include/irr/asset/filters/CMatchedSizeInOutImageFilterCommon.h

```diff
@@ -139,7 +139,7 @@ class CMatchedSizeInOutImageFilterCommon : public CBasicImageFilterCommon
 				// setup convert state
 				// I know my two's complement wraparound well enough to make this work
 				const auto& outRegionOffset = commonExecuteData.oit->imageOffset;
-				commonExecuteData.offsetDifference = state->outOffsetBaseLayer-(core::vectorSIMDu32(outRegionOffset.x,outRegionOffset.y,outRegionOffset.z,commonExecuteData.oit->imageSubresource.baseArrayLayer)+state->inOffsetBaseLayer);
+				commonExecuteData.offsetDifference = core::vectorSIMDi32(state->outOffsetBaseLayer.x, state->outOffsetBaseLayer.y, state->outOffsetBaseLayer.z, state->outOffsetBaseLayer.w)-(core::vectorSIMDi32(outRegionOffset.x,outRegionOffset.y,outRegionOffset.z,commonExecuteData.oit->imageSubresource.baseArrayLayer)+state->inOffsetBaseLayer);
```

Comment:
no need to change types to `i32`, see the comment above.

Reply:
corrected in following commit